xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserSheet.xml

```diff
@@ -448,17 +448,21 @@ return XWiki;
   width: 13em;
 }
 
-#avatar p {
-  text-align: center;
-}
+#avatar {
+  p {
+    text-align: center;
+    }
+
+  img{
+    border-radius: @border-radius-base @border-radius-base 0 0;
```

Comment:
Shouldn't we have the border radius on all corners with the way the UI looks now?

Reply:
If we want to have both elements visually integrated it is necessary to remove the radius. I could have them on, but then a white space would be needed to keep visual balance.

<img width="215" alt="Screenshot 2024-02-14 at 09 58 22" src="https://github.com/xwiki/xwiki-platform/assets/149672322/3789e3b7-8c15-4ce6-832a-a4dc9e81d269">

Reply:
Makes sense with your latest change :+1:

Reply:
Can I mark this as resolved?

Reply:
Yup :+1: